src/messages/MessageBuilder.hpp

```diff
@@ -68,6 +72,9 @@ class MessageBuilder
     }
 
 private:
+    TextElement *emplaceSystemTextAndUpdate(const QString &text,
```

Comment:
Could you write a comment clarifying what this does?

Reply:
Added in 44263e7